src/forwarddiff.jl

```diff
@@ -1,9 +1,49 @@
+"""
+promote_dual(::Type{T},::Type{T2})

```

Comment:
```suggestion
    promote_dual(::Type{T},::Type{T2})

```
For correct highlighting as code